mesheryctl/internal/cli/root/config/config.go

```diff
@@ -256,6 +257,16 @@ func (ctx *Context) ValidateVersion() error {
 	return nil
 }
 
+// GetEnforcedProvider returns the enforced provider of the current context
+func (ctx *Context) GetEnforcedProvider() string {
+	return ctx.EnforcedProvider
+}
+
+// SetEnforcedProvider sets the enforced provider of the current context
+func (ctx *Context) SetEnforcedProvider(enforcedProvider string) {
+	ctx.EnforcedProvider = enforcedProvider
```

Comment:
We should be safe to drop the “Enforced” portion of these names, leaving just “Provider”.

Reply:
Done

Reply:
@Revolyssup please note.